### PR TITLE
Feature/1428

### LINKED ISSUE #1428: Web Notification front end [Desktop]
As a commentor/author of a post, I want to receive a notification for every comment posted, as well as when a post or comment is edited.
Current Scenario: When someone comments on my post, I don't know right away because I'm not sent an email or notification.

Expected results: As soon as someone comments on my post or likes/shares it, I should see a web notification to alert me, (similar to notifications on facebook or reddit)

Initial design: https://fightpandemics.slack.com/archives/C018ZNAD62Y/p1598344009136700 

image

### PR DESCRIPTION
**Pull request by Serhazor**
### What does this pull request aim to achieve? 💯
solves #1428 

Adelina will work on styling and data for it.

_Please be concise and link any related issue(s):_

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix. The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Comment by kterry1** [comment]
Changed to a Draft since it's still in progress.

File: client/.env [hunk 1]
```diff
@@ -1 +1,11 @@
-EXTEND_ESLINT=true
```
Comment (mannykary) [inline client/.env:1]: @Serhazor Please be very careful in the future - it looks like you have accidentally committed secrets to git. I am going to go ahead and rebase to remove these secrets from the git history. Also, these environment variables are for the backend `.env`, not the frontend.

**Comment by mannykary** [comment]
@Serhazor I rebased and force pushed to this branch to remove secrets that were accidentally committed.

**Comment by Zaydme** [comment]
@mannykary @ddveloper  please check this PR along with https://github.com/FightPandemics/FightPandemics/issues/1428
maybe they should be closed?

**Comment by ddveloper** [comment]
> @mannykary @ddveloper please check this PR along with #1428
> maybe they should be closed?

@Zaydme I closed #1428 since I merged all the change into this PR

**Pull request closed without merging**